src/App.tsx

```diff
@@ -1,18 +1,29 @@
-import { PeoplePage } from './components/PeoplePage';
-import { Navbar } from './components/Navbar';
+import { Routes, Route, Navigate } from 'react-router-dom';
+
+import { PeoplePage } from './components/PeoplePage/PeoplePage';
+import { MainNav } from './components/MainNav';
+import { NotFoundPage } from './components/NotFoundPage';
 
 import './App.scss';
 
 export const App = () => {
   return (
     <div data-cy="app">
-      <Navbar />
+      <MainNav />
 
       <div className="section">
         <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
+          <Routes>
+            <Route path="/" element={<h1 className="title">Home Page</h1>} />
```

Comment:
It's better to have some kind of variable that stores all route values in one place, enum, or just a const. It'll be very clear to see all existing routes in one place and also when in some other components you have to use a redirect or link you can import that variable and avoid using hardcoded values which usually cause bugs and makes refactoring harder.